README.md

```diff
@@ -241,7 +241,7 @@ You can perform cross-platform builds. For example, build and assemble a Windows
 ```bash
 export JAVA_HOME=$(/usr/libexec/java_home) # required by OpenSearch install-plugin during assemble
 ./build.sh manifests/1.1.0/opensearch-1.1.0.yml --snapshot --platform windows
```

Comment:
in this doc, some version path use 1.1.0, some use 1.2.0, could we keep consistency